homeassistant/components/honeywell/climate.py

```diff
@@ -95,8 +96,8 @@ class HoneywellUSThermostat(ClimateEntity):
 
     def __init__(self, data, device, cool_away_temp, heat_away_temp):
         """Initialize the thermostat."""
-        self._data = data
-        self._device = device
+        self._data: HoneywellData = data
```

Comment:
Please type the entity signature instead.

Reply:
I didn't intend to put this change in this branch... but since it's here we'll keep it